plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/converter/GlueInputConverter.java

```diff
@@ -64,10 +67,14 @@ public static DatabaseInput convertDatabase(Database database)
 
     public static TableInput convertTable(Table table)
     {
-        Optional<String> comment = Optional.ofNullable(table.getParameters().get(TABLE_COMMENT));
-        Map<String, String> tableParameters = table.getParameters().entrySet().stream()
-                .filter(entry -> !entry.getKey().equals(TABLE_COMMENT))
-                .collect(toImmutableMap(Map.Entry::getKey, Map.Entry::getValue));
+        Map<String, String> tableParameters = table.getParameters();
+        Optional<String> comment = Optional.empty();
+        if (!isTrinoView(table) && !isTrinoMaterializedView(table)) {
+            tableParameters = tableParameters.entrySet().stream()
+                    .filter(entry -> !entry.getKey().equals(TABLE_COMMENT))
+                    .collect(toImmutableMap(Entry::getKey, Entry::getValue));
+            comment = Optional.ofNullable(table.getParameters().get(TABLE_COMMENT));
```

Comment:
nit: reverse & reuse variable

```suggestion
            comment = Optional.ofNullable(tableParameters.get(TABLE_COMMENT));
            tableParameters = tableParameters.entrySet().stream()
                    .filter(entry -> !entry.getKey().equals(TABLE_COMMENT))
                    .collect(toImmutableMap(Entry::getKey, Entry::getValue));
```